services/api/src/typeDefs.js

```diff
@@ -606,6 +606,10 @@ const typeDefs = gql`
     """
     organization: Int
     """
+    MetaData of organization
+    """
+    organizationMetaData: OrganizationMetaData
```

Comment:
Blaize and I had a chat, we think this should change to return the full `Organization` type, effectively removing the need for this new type.

Then in the `getMinimalOrganizationByProject` resolver, add a check that will check if any other fields than the following are requested, then a `hasPermission` check against the organization id would be performed.

Here is an example of how to check if a field was requested https://github.com/uselagoon/lagoon/blob/v2.25.0/services/api/src/resources/env-variables/resolvers.ts#L100

So effectively you would do something like this in the resolver
```
if fields notin id, name, friendlyName, description {
  hasPermission
end
```